src/public/uploadHandler.js

```diff
@@ -0,0 +1,33 @@
+/* global Toastify */
+document.addEventListener('DOMContentLoaded', function () {
+  const form = document.querySelector('form');
+  const userId = document.querySelector('#userId').value;
+
+  form.addEventListener('submit', async function (event) {
+    event.preventDefault();
+
+    const formData = new FormData(form);
+    try {
+      const response = await fetch(
+        '/api/v1/users/' + userId + '/uploadProfile',
+        {
+          method: 'POST',
+          body: formData,
+        }
+      );
+
+      if (response.ok) {
+        const data = await response.json();
+
```

Comment:
currently you're not doing anything with the `imageUrl` that should be returned in the response. 

You can use DOM functions to append an `img` element to the page when the response is successful, and set the `src` attribute of that element to the imageUrl that the server returned.

You may need to update the `app.use('helmet')` line in Express to tell the server that it's okay to show images from `imgur` (cross-site content):

```
app.use(helmet({
  contentSecurityPolicy: {
    directives: {
      "img-src": ["'self'", "https://i.imgur.com"]
    }
  }
}));
```

Reply:
could you please check?
 commit  acaff4dbd58217f9489eabf637b86280f11d49ab